src/proto/parameters.proto

```diff
@@ -413,12 +413,19 @@ message ShootOrPassPlayConfig
 
 message RobotCapabilitiesConfig
 {
-    // Comma-separated list of numbers of robots with broken dribblers
-    required string broken_dribblers = 1;
-    // Comma-separated list of numbers of robots with broken chippers
-    required string broken_chippers = 2;
-    // Comma-separated list of numbers of robots with broken kickers
-    required string broken_kickers = 3;
+    required BrokenRobots broken_dribblers = 1;
+    required BrokenRobots broken_chippers  = 2;
+    required BrokenRobots broken_kickers   = 3;
+}
+
+message BrokenRobots
+{
+    required bool robot_0 = 1;
+    required bool robot_1 = 2;
+    required bool robot_2 = 3;
+    required bool robot_3 = 4;
+    required bool robot_4 = 5;
+    required bool robot_5 = 6;
```

Comment:
The number of robots will change when we move from div B to div A, so it would be better to consider a more flexible approach. Could we instead use a `repeated uint32` field that stores the IDs of the robots that are broken?

Reply:
@wmostrenko do you remember what the outcome of this discussion was. I vaguely remember talking about it during software-general